### ISSUE TITLE
WCAG 2.2: Nine new issues need documenting

### ISSUE THREAD

**Issue by estelle** [mdn/content#26831]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG

### What specific section or headline is this issue about?

Nine a11y success criteria have been updated and need to be documented

### What information was incorrect, unhelpful, or incomplete?

The following needs to be documented:

 [Guideline 2.4 Navigable](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#guideline-24-navigable)

-  [2.4.11 Focus Not Obscured (Minimum) (AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2411-focus-not-obscured-minimum-aa)
-  [2.4.12 Focus Not Obscured (Enhanced) (AAA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2412-focus-not-obscured-enhanced-aaa)
-  [2.4.13 Focus Appearance (AAA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2413-focus-appearance-aaa)

[Guideline 2.5 Input Modalities](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#guideline-25-input-modalities)

- [2.5.7 Dragging Movements (AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#257-dragging-movements-aa)
- [2.5.8 Target Size (Minimum) (AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#258-target-size-minimum-aa)

[Guideline 3.2 Predictable](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#guideline-32-predictable)

- [3.2.6 Consistent Help (A)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#326-consistent-help-a)

[Guideline 3.3 Input Assistance](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#guideline-33-input-assistance)

-  [3.3.7 Redundant Entry (A)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#337-redundant-entry-a)
-   [3.3.8 Accessible Authentication (Minimum) (AA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#338-accessible-authentication-minimum-aa)
-   [3.3.9 Accessible Authentication (Enhanced) (AAA)](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#339-accessible-authentication-enhanced-aaa)

### What did you expect to see?

someone working on this

**Comment by schalkneethling** [comment]
As mentioned on Slack, I would LOVE to help with this.

**Comment by CBID2** [comment]
> As mentioned on Slack, I would LOVE to help with this.

I'd love to help as well @schalkneethling &  @estelle! :)

**Comment by estelle** [comment]
If either or both of you would like to work on the project, please assign to yourself (or can you not do that via the UI?).

When added, the  WCAG landing page https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG should also be updated to reflect the new criteria

**Comment by CBID2** [comment]
> If either or both of you would like to work on the project, please assign to yourself (or can you not do that via the UI?).
> 
> 
> 
> When added, the  WCAG landing page https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG should also be updated to reflect the new criteria

I can't assign myself
